lib/aur-chroot

```diff
@@ -113,8 +113,9 @@ if ((prepare)); then
 fi
 
 if ((build)); then
-    sudo PKGDEST="${PKGDEST:-$PWD}" makechrootpkg -r "$directory" \
-         "${makechrootpkg_bindmounts_ro[@]}" "${makechrootpkg_args[@]}"
+    sudo --preserve-env=GNUPGHOME env PKGDEST="${PKGDEST:-$PWD}" \
```

Comment:
The man page mentions that preserve-env takes a comma-separated list (seems a recent introduction), so
```
sudo --preserve-env=GNUPGHOME,PKGDEST
```
should work since makechrootpkg [defaults to PWD](https://git.archlinux.org/devtools.git/tree/makechrootpkg.in#n386) if PKGDEST is not set.

Reply:
This is the best solution then

Reply:
I overlooked this, thanks. Now this is truly a one-line change =)